pkg/applicationserver/io/packages/redis/registry.go

```diff
@@ -230,8 +230,7 @@ func (r ApplicationPackagesRegistry) SetAssociation(ctx context.Context, ids *tt
 				pb = &ttnpb.ApplicationPackageAssociation{}
 			}
 
-			now := time.Now().UTC()
-			pb.UpdatedAt = &now
+			pb.UpdatedAt = ttnpb.ProtoTimePtr(time.Now())
```

Comment:
Here is isn't. Any reason for that?

Reply:
Proto time is just Unix time with nanosecond precision. It doesn't carry timezone info, so `UTC()` jost isn't necessary. I've also removed it from the places where I didn't remove it yet.